nodejs/nodejs.lua

```diff
@@ -16,18 +16,33 @@ VersionSourceUrl = "https://nodejs.org/dist/index.json"
 PLUGIN = {
     name = "nodejs",
     author = "Aooohan",
-    version = "0.0.3",
+    version = "0.0.4",
     description = "Node.js",
     updateUrl = "https://raw.githubusercontent.com/version-fox/version-fox-plugins/main/nodejs/nodejs.lua",
 }
 
+local is_debug = os.getenv("DEBUG")
+
 function PLUGIN:PreInstall(ctx)
     local version = ctx.version
 
     if version == "latest" then
-        local lists = self:Available({})
-        version = lists[1].version
+        local result = self:Available({})
+        version = result['list'][1].version
+    end
+
+    if not is_semver_simple(version) then
+        local result = self:Available({})
+        version = result['versions_shorthand'][version]
+        if is_debug then
```

Comment:
if you think remove the debug statement is better, please tell me

Reply:
It may be planned later to uniformly handle this kind of debug flag. It may not be obtained through environment variables. I haven't thought about it yet, so let's keep it  for now.